### PR TITLE
III-3776 Switch to PHP-CS-Fixer

### PR DESCRIPTION
**Pull request by lucwollants**
**Note: Deleted to work around hanging GitHub Actions**

### Added
- Add strict type declaration
- Marked classes as final

### Changed
- Switched from `squizlabs/php_codesniffer` to `publiq/php-cs-fixer-config`

### Removed
- Removed `satooshi/php-coveralls`
 
---
Ticket: https://jira.uitdatabank.be/browse/III-3776

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
             - name: ✅ Run tests
               run: composer test
 
-    phpcs:
+    cs:
```
Comment (jdrieghe) [inline .github/workflows/ci.yml:30]: @bertramakers will have to change the required CI steps in the Github settings then

**Review by jdrieghe — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 1 above)
Comment (lucwollants) [reply .github/workflows/ci.yml:30]: Could that also be the reason why the GitHub Actions are hanging?

Comment (bertramakers) [reply .github/workflows/ci.yml:30]: `Code style` is already/still enabled. I think this has to be merged first so the reference to `phpcs` gets removed on the main branch.

Comment (jdrieghe) [reply .github/workflows/ci.yml:30]: Ah no, it looks like it uses the `name`. Awesome.

Comment (bertramakers) [reply .github/workflows/ci.yml:30]: Okay!

**Pull request closed without merging**